dynomitemanager-core/src/main/java/com/netflix/dynomitemanager/config/FloridaConfig.java

```diff
@@ -281,9 +280,20 @@
     // Storage engine: ARDB with RocksDB
     // =================================
 
+    /**
+     * Compaction strategy for RocksDB. RocksDB allows for optimized compaction
+     * strategies: OptimizeLevelStyleCompaction,
+     * OptimizeUniversalStyleCompaction or none.
+     * 
+     * @return the compaction strategy
+     */
+    @DefaultValue("none")
+    @PropertyName(name = "dyno.ardb.rocksdb.compactionStrategy")
+    public String getRocksDBCompactionStrategy();
+
     @DefaultValue("128")
     @PropertyName(name = "dyno.ardb.rocksdb.writebuffermb")
-    public int getWriteBufferSize();
+    public int getRocksDBWriteBufferSize();
```

Comment:
getARDBRocksDBWriteBufferSize

Reply:
this is a RocksDB configuration. I am trying to differentiate between RocksDB and ARDB configurations.